docs/guides/universal-receiver-delegate/set-default-implementation.md

```diff
@@ -369,7 +369,7 @@ Lastly, we need to send the transaction that will update the URD and its permiss
 
 ```typescript title="Update the data on the Universal Profile"
 // update the Universal Profile data
-await universalProfile.methods['setData(bytes32[],bytes[])'](
+await universalProfile.methods.setData(
```

Comment:
this is incorrect. The function is `setDataBatch` here with v0.10.0.

This applies to all the other changes made in this page.

```suggestion
await universalProfile.methods.setDataBatch(
```